http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -246,6 +246,29 @@ class ProxyRoutesSpec
     result.futureValue shouldBe "Hello Leonardo!"
   }
 
+  it should "create terminal trying to access a terminal not created yet" in {
```

Comment:
Should this be something like

```suggestion
  it should "create the terminal when trying to access a terminal that was not created yet" in {
```

Reply:
I just moved this test because it was in the wrong place.. but yeah, that name sounds better. Will fix.